public/scripts/controllers/homeController.js

```diff
@@ -0,0 +1,10 @@
+(function (module) {
+const homeController ={};
+
+homeController.initHomeIndex = function () {
+  Chicken.fetchAll();
```

Comment:
Each time I go to About and back to Home, another set of chickens is added to the existing ones, repeating them all over and over...  Perhaps doing a fetchAll is not the right call here?